### ISSUE TITLE
Crash when open Gadomancy tab on creative

### ISSUE THREAD

**Issue by ralthasgames** [makeoo/Gadomancy#12]
---- Minecraft Crash Report ----
// You're mean.

Time: 10/27/15 2:39 PM
Description: Rendering item

java.lang.NoClassDefFoundError: net/minecraft/src/FMLRenderAccessLibrary
    at makeo.gadomancy.client.renderers.item.ItemRenderRemoteJar.renderItem(ItemRenderRemoteJar.java:51)
    at net.minecraftforge.client.ForgeHooksClient.renderInventoryItem(ForgeHooksClient.java:183)
    at net.minecraft.client.renderer.entity.RenderItem.func_82406_b(RenderItem.java:563)
    at net.minecraft.client.gui.inventory.GuiContainer.func_146977_a(GuiContainer.java:271)
    at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:99)
    at net.minecraft.client.renderer.InventoryEffectRenderer.func_73863_a(InventoryEffectRenderer.java:38)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_73863_a(GuiContainerCreative.java:638)
    at net.minecraft.client.renderer.EntityRenderer.func_78480_b(EntityRenderer.java:1061)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1001)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:310)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:394)
    at org.multimc.EntryPoint.listen(EntryPoint.java:170)
    at org.multimc.EntryPoint.main(EntryPoint.java:54)
Caused by: java.lang.ClassNotFoundException: net.minecraft.src.FMLRenderAccessLibrary
    at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
    at java.lang.ClassLoader.loadClass(Unknown Source)
    at java.lang.ClassLoader.loadClass(Unknown Source)
    ... 25 more
Caused by: java.lang.NullPointerException
## A detailed walkthrough of the error, its code path and all known details is as follows:

-- Head --
Stacktrace:
    at makeo.gadomancy.client.renderers.item.ItemRenderRemoteJar.renderItem(ItemRenderRemoteJar.java:51)
    at net.minecraftforge.client.ForgeHooksClient.renderInventoryItem(ForgeHooksClient.java:183)

-- Item being rendered --
Details:
    Item Type: makeo.gadomancy.common.items.ItemBlockRemoteJar@1490b1a4
    Item Aux: 0
    Item NBT: null
    Item Foil: false
Stacktrace:
    at net.minecraft.client.renderer.entity.RenderItem.func_82406_b(RenderItem.java:563)
    at net.minecraft.client.gui.inventory.GuiContainer.func_146977_a(GuiContainer.java:271)
    at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:99)
    at net.minecraft.client.renderer.InventoryEffectRenderer.func_73863_a(InventoryEffectRenderer.java:38)
    at net.minecraft.client.gui.inventory.GuiContainerCreative.func_73863_a(GuiContainerCreative.java:638)

-- Screen render details --
Details:
    Screen name: net.minecraft.client.gui.inventory.GuiContainerCreative
    Mouse location: Scaled: (186, 46). Absolute: (747, 831)
    Screen size: Scaled: (480, 255). Absolute: (1920, 1017). Scale factor of 4

-- Affected level --
Details:
    Level name: MpServer
    All players: 1 total; [EntityClientPlayerMP['ralthas'/32, l='MpServer', x=-1764.50, y=5.62, z=-560.50]]
    Chunk stats: MultiplayerChunkCache: 555, 555
    Level seed: 0
    Level generator: ID 01 - flat, ver 0. Features enabled: false
    Level generator options: 
    Level spawn location: World: (-1770,4,-558), Chunk: (at 6,0,2 in -111,-35; contains blocks -1776,0,-560 to -1761,255,-545), Region: (-4,-2; contains chunks -128,-64 to -97,-33, blocks -2048,0,-1024 to -1537,255,-513)
    Level time: 117 game time, 117 day time
    Level dimension: 0
    Level storage version: 0x00000 - Unknown?
    Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
    Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
    Forced entities: 20 total; [EntityClientPlayerMP['ralthas'/32, l='MpServer', x=-1764.50, y=5.62, z=-560.50], EntityVillager['Villager'/6, l='MpServer', x=-1760.88, y=4.00, z=-524.60], EntityVillager['Villager'/7, l='MpServer', x=-1759.50, y=5.00, z=-487.50], EntityVillager['Villager'/10, l='MpServer', x=-1743.50, y=5.00, z=-519.50], EntityVillager['Villager'/11, l='MpServer', x=-1743.50, y=5.00, z=-506.50], EntityVillager['Villager'/12, l='MpServer', x=-1743.50, y=5.00, z=-505.50], EntityBat['Bat'/2732, l='MpServer', x=-1723.50, y=5.87, z=-490.52], EntityVillager['Villager'/16, l='MpServer', x=-1735.50, y=5.00, z=-518.50], EntityVillager['Villager'/17, l='MpServer', x=-1735.50, y=5.00, z=-501.50], EntityVillager['Villager'/18, l='MpServer', x=-1735.50, y=5.00, z=-500.50], EntityVillager['Villager'/19, l='MpServer', x=-1730.50, y=5.00, z=-487.50], EntityVillager['Villager'/20, l='MpServer', x=-1729.50, y=5.00, z=-487.50], EntityVillager['Villager'/22, l='MpServer', x=-1720.50, y=5.00, z=-486.50], EntityVillager['Villager'/23, l='MpServer', x=-1714.68, y=5.00, z=-525.50], EntityVillager['Villager'/24, l='MpServer', x=-1715.49, y=4.69, z=-524.60], EntityVillager['Villager'/25, l='MpServer', x=-1716.50, y=4.00, z=-494.50], EntityVillager['Villager'/26, l='MpServer', x=-1719.50, y=5.00, z=-516.50], EntityVillager['Villager'/27, l='MpServer', x=-1712.50, y=5.00, z=-507.50], EntityVillager['Villager'/28, l='MpServer', x=-1712.50, y=5.00, z=-506.50], EntityVillager['Villager'/29, l='MpServer', x=-1712.50, y=5.00, z=-485.50]]
    Retry entities: 0 total; []
    Server brand: fml,forge
    Server type: Integrated singleplayer server
Stacktrace:
    at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:373)
    at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2444)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:919)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:310)
    at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:394)
    at org.multimc.EntryPoint.listen(EntryPoint.java:170)
    at org.multimc.EntryPoint.main(EntryPoint.java:54)

-- System Details --
Details:
    Minecraft Version: 1.7.10
    Operating System: Windows 10 (amd64) version 10.0
    Java Version: 1.8.0_65, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 390401456 bytes (372 MB) / 722468864 bytes (689 MB) up to 954728448 bytes (910 MB)
    JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx1024m
    AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1448 6 mods loaded, 6 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCHIJAAAA   mcp{9.05} [Minecraft Coder Pack](minecraft.jar) 
    UCHIJAAAA   FML{7.10.99.99} [Forge Mod Loader](forge-1.7.10-10.13.4.1448-1.7.10-universal.jar) 
    UCHIJAAAA   Forge{10.13.4.1448} [Minecraft Forge](forge-1.7.10-10.13.4.1448-1.7.10-universal.jar) 
    UCHIJAAAA   Baubles{1.0.1.10} [Baubles](Baubles-1.7.10-1.0.1.10.jar) 
    UCHIJAAAA   Thaumcraft{4.2.3.5} [Thaumcraft](Thaumcraft-1.7.10-4.2.3.5.jar) 
    UCHIJAAAA   gadomancy{1.0.0} [Gadomancy](gadomancy-1.7.10-1.0.1.jar) 
    GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 358.50' Renderer: 'GeForce GTX 970/PCIe/SSE2'
    Launched Version: MultiMC5
    LWJGL: 2.9.1
    OpenGL: GeForce GTX 970/PCIe/SSE2 GL version 4.5.0 NVIDIA 358.50, NVIDIA Corporation
    GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

```
Is Modded: Definitely; Client brand changed to 'fml,forge'
Type: Client (map_client.txt)
Resource Packs: []
Current Language: English (US)
Profiler Position: N/A (disabled)
Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
Anisotropic Filtering: Off (1)
```

**Comment by makeo** [comment]
Thanks for the detailed report! Bug will be fixed next version (in 1 or 2 Days)
Fix: https://github.com/makeoo/Gadomancy/commit/8e147dde099289f5b57964db21c89dbb83f68917

**Comment by makeo** [comment]
Fixed with version 1.0.2
